audit/auditLog.json

```diff
@@ -596,6 +611,12 @@
       "1.0.1": ["audit20250109_3"],
       "1.1.0": ["audit20250109_2", "audit20250508"]
     },
+    "LiFiTimelockController": {
```

Comment:
This PR is erroneously overwriting the LiFiTimelockController audit entry in an incomplete form (1.0.1 is missing)

Reply:
fixed